Source/ALDataRequestView.swift

```diff
@@ -197,6 +233,11 @@ public class ALDataRequestView: UIView {
         emptyView = dataSourceEmptyView
         addSubview(emptyView!)
         emptyView?.autoPinEdgesToSuperviewEdges()
+        
+        dataSourceEmptyView.alpha = 0
+        UIView.animateWithDuration(dataSource?.showAnimationDurationForDataRequestView(self) ?? 0, animations: {
+            dataSourceEmptyView.alpha = 1
+        })
```

Comment:
zelfde verhaal als iets meer naar boven. Bovendien duplicate code, misschien samenvoegen in een showView achtige methode?